schema/json/vnd.syft+json

```diff
@@ -0,0 +1,70 @@
+(registered 2021-11-05, last updated 2021-11-05)
+
+Media type name: application
+Media subtype name: vnd.syft+json
+
+Required parameters: N/A
+
+Optional parameters:
+version
+
+The version parameter refers to the Syft specification version in use.
+
+version = 1*DIGIT "." 1*DIGIT "." 1*DIGIT
```

Comment:
Question: does `1*DIGIT` only allow for values like `5` and not `50`?

Reply:
Good Question - here is the relevant text.
https://www.rfc-editor.org/rfc/rfc2045.html#section-4

I asked the reviewer if we could use `x.x.x` and they said it was fine extending to M/m/p here.

MIME-VERSION in their specification does only allow for a single digit

```
version := "MIME-Version" ":" 1*DIGIT "." 1*DIGIT
```

I can follow up and see what edits we might need for the submission since this is unclear.

Reply:
I'll make a follow-up PR when I get more clarity from the governing body. As of right now, they accepted the proposal as is. Edits are welcome should this turn out to be more strict than we need for the version constraints.